ci/code_checks.sh

```diff
@@ -109,9 +109,12 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
 
     # Imports - Check formatting using isort see setup.cfg for settings
     MSG='Check import format using isort ' ; echo $MSG
-    isort --recursive --check-only pandas asv_bench
+    if [[ "$AZURE" == "true" ]]; then
+        isort --recursive --check-only pandas asv_bench | awk -F ':' '{print $0"\n" "##vso[task.logissue type=error;sourcepath=%(path)s]%(text)s"}'
```

Comment:
I think the return code `$?` we are checking later to see if `isort` was successful or not, will actually be the one of awk with this, which will always be 0.

This means that if the imports are not correctly sorted, we won't detect them in the CI.

Can you start by swapping the order of two imports in pandas in this PR, so we know if this is working or not? We'll remove them once the PR is approved.

Reply:
Yes, I will push up the badly sorted imports

Reply:
here is the output! ```Check import format using isort
ERROR: /home/vsts/work/1/s/pandas/plotting/_matplotlib/core.py Imports are incorrectly sorted.
##[error]%(path)s(,): error : %(text)s
Check import format using isort DONE
##[error]Bash exited with code '1'.
##[section]Finishing: Linting```